news/2022-05-20-beatmap-spotlights-season-6-spring-2022-high-speed.md

```diff
@@ -0,0 +1,283 @@
+
+### osu!taiko
+
+#### Curators' Pick
+
+[mafumafu - Hello Dystopia [Collapsing Reality]](https://osu.ppy.sh/beatmapsets/809048#taiko/1697683), mapped by [JarvisGaming](https://osu.ppy.sh/users/8601048)
+
+Coming in hot this season's osu!taiko Spotlights, we have JarvisGaming's rendition of mafumafu's Hello Dystopia!
+
+His emblematic "Collapsing Reality" difficulty is an exemplary demonstration of both structuring and patterning, with notes carefully arranged to fit the song like a glove yet still being fun and entertaining for the player. Ranging from the faster chorus sections down to the break-off & deceleration in the 2nd half of the theme, it all climaxes into an extraordinarily grandiloquent experience.
+
+And yes, the top diff still slaps really damn hard.
+
+*described and primarily suggested by [Axer](https://osu.ppy.sh/users/7299864)*
+
+#### Players' Pick
+
+[TUYU - Daemonisch [KTYN's Inner Oni]](https://osu.ppy.sh/beatmapsets/1444402#taiko/2983308), mapped by [KTYN](https://osu.ppy.sh/users/2250574)
+
+Arriving at the demand of player's choice, we have KTYN's interpretation of TUYU's Daemonisch, hosted by certified TUYU enjoyer gaston_2199!
+
+KTYN's Inner Oni proves that simplicity can be just as striking as catharsis, with a structure that is simplistic when necessary, getting more frantic as the song progresses, and culminating into a grand finale that spikes the density with carefully crafted piano streams. This is a map that can be enjoyed by a wide audience (as long as you can survive it, that is!).
+
+*primarily suggested by Nepuri*
+
+*described by [Axer](https://osu.ppy.sh/users/7299864)*
+
+### osu!catch
+
+#### Curators' Pick
+
+[Chroma - Made In Love [Rei's MAXIMUM]](https://osu.ppy.sh/beatmapsets/1028944#fruits/2151543), mapped by [Bunnrei](https://osu.ppy.sh/users/829284)
+
+There are few maps in osu!catch I can think of that is mapped to the song as well as Bunnrei's MAXIMUM for Made in Love.
+
+The map masterfully manages to convey all of the distinct parts of the song in a way that highlights each section's own distinct theme. Whether it be the less dense introduction, the more technical and synth heavy middle, or the denser streams mapped near the end, everything is mapped in a way that highlights noises and themes in the song.
+
+While its not the most technical or difficult map that Bunnrei has ever mapped, this map is one that any experienced fruit catcher can enjoy for themselves.
+
+*described and primarily suggested by [wonjae](https://osu.ppy.sh/users/5032045)*
+
+#### Players' Pick
+
+[SHK - Halloween Party [Juansy's Rain]](https://osu.ppy.sh/beatmapsets/1609415#fruits/3286191), mapped by [Juansy-](https://osu.ppy.sh/users/12588215)
+
+Though it's either too late or too early into the Halloween you can't deny the fact that this diff on this set is comfortable enough to play either for the casual fruit fan or to the hardcore maniac catch enjoyer.
```

Comment:
```suggestion
Although it may be either *slightly* too late or too early into Halloween, you can't deny the fact that this diff on this set is comfortable enough to play either for the casual fruit fan or to the hardcore maniac catch enjoyer.
```